examples/postcard.py

```diff
@@ -25,16 +25,6 @@
     address_zip='12345'
 )
 
-print("\n")
```

Comment:
Is there a reason we're removing this section? It seems like the goal is to show the output every time we hit the Lob API and get a response back

Reply:
We're already showing the address output in the `to` and `from` sections, and there are `verify` examples, so I felt like it would be redundant to show it again.